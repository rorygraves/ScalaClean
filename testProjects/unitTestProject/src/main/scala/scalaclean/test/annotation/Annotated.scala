package scalaclean.test.annotation

@BasicJava
@ComplexJava(intValue = 1, doubleValue = 2, stringValue = "XX", booleanValue = true)
@BasicScala
@ComplexScala(1, 2, "yy", true)
class annotated {

  @BasicJava
  @ComplexJava(intValue = 1, doubleValue = 2, stringValue = "XX", booleanValue = true, intValueWithDefault = 99)
  @BasicScala
  @ComplexScala(1, 2, "yy", true, 99)
  def foo = 0

}
