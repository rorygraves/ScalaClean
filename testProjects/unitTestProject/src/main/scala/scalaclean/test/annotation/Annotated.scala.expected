package scalaclean.test.annotation

@BasicJava
@ComplexJava(intValue = 1, doubleValue = 2, stringValue = "XX", booleanValue = true)
@BasicScala
@ComplexScala(1, 2, "yy", true)
class annotated {

  @BasicJava
  @ComplexJava(intValue = 1, doubleValue = 2, stringValue = "XX", booleanValue = true, intValueWithDefault = 99)
  @BasicScala
  @ComplexScala(1, 2, "yy", true, 99)
  def foo = 0/* annotations(C:scalaclean.test.annotation.annotated/M:foo()) - AnnotationData[-164,-55,scalaclean.test.annotation.ComplexJava,Map(booleanValue -> true, doubleValue -> 2.0, intValue -> 1, intValueWithDefault -> 99, stringValue -> XX)] :: AnnotationData[-177,-168,scalaclean.test.annotation.BasicJava,Map()] :: AnnotationData[-37,-3,scalaclean.test.annotation.ComplexScala,Map(0 -> 1, 1 -> 2.0, 2 -> yy, 3 -> true, 4 -> 99)] :: AnnotationData[-51,-41,scalaclean.test.annotation.BasicScala,Map()] */

}/* annotations(C:scalaclean.test.annotation.annotated) - AnnotationData[-128,-45,scalaclean.test.annotation.ComplexJava,Map(booleanValue -> true, doubleValue -> 2.0, intValue -> 1, stringValue -> XX)] :: AnnotationData[-139,-130,scalaclean.test.annotation.BasicJava,Map()] :: AnnotationData[-31,-1,scalaclean.test.annotation.ComplexScala,Map(0 -> 1, 1 -> 2.0, 2 -> yy, 3 -> true, 4 -> <<<TODO>>>)] :: AnnotationData[-43,-33,scalaclean.test.annotation.BasicScala,Map()] */
